lib/frontend/components/GradientButton.dart

```diff
@@ -0,0 +1,67 @@
+import 'package:flutter/material.dart';
+import 'dart:math' as math;
+
+class GradientButton extends StatelessWidget {
+  final String buttonText;
+  final Function buttonPressed;
+
+  static final LinearGradient gradientOutline = const LinearGradient(
+    begin: Alignment.topLeft,
+    end: Alignment.topRight,
+    colors: const [Color(0xFFFF6C1A), Color(0xFFF01844), Color(0xFF7E0BC9)],
+    stops: const [0.0, .528, 1.0],
+    transform: GradientRotation(math.pi/4),
+  );
+
+  final innerDecor = BoxDecoration(
+    color: Color(0xFF121212),
+      border: Border.all(color: Colors.transparent),
+    borderRadius: BorderRadius.circular(50)
+  );
+
+  final gradientBoxDecoration = BoxDecoration(
+    gradient: gradientOutline,
+    borderRadius: BorderRadius.circular(50),
+  );
+
+  GradientButton({
+    @required this.buttonText,
+    @required this.buttonPressed
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.only(top: 20.0),
+      child: Padding(
+        padding: const EdgeInsets.all(2.0),
+        child: Container(
+          child: ButtonTheme(
+            child:FlatButton(
+              color: Colors.transparent,
+              onPressed: buttonPressed,
+              shape: RoundedRectangleBorder(
+                  borderRadius: new BorderRadius.circular(50)
```

Comment:
You do not need to explicitly use the new keyword when instantiating new objects in new versions of Dart.  Another unimportant detail, so if the new keyword helps you understand then keep it.